setup.py

```diff
@@ -1,19 +1,28 @@
 import os
-from setuptools import setup, Extension
-from setuptools.command.build_ext import build_ext
 import sys
+
+import numpy as np
```

Comment:
Dependencies sorted in all Python code according to PEP8 https://www.python.org/dev/peps/pep-0008/#imports

(I ran isort)